### PR TITLE
domains -> keys

### PR DESCRIPTION
**Pull request by tianhuil**
Small tweaks to change domains -> keys so that variable names are sufficiently general.

### CODE REVIEW

**Comment by tianhuil** [comment]
@dangra: would love to merge and deploy to pypi so that we can merge https://github.com/scrapy/scrapy/pull/3160.  The [current Travis error](https://travis-ci.org/scrapy/scrapy/jobs/351109062) on https://github.com/scrapy/scrapy/pull/3160 is caused by this latest version of queuelib not being up on pypi.

**Comment by dangra** [comment]
@tianhuil thanks for the heads up. It is in pypi now.

File: queuelib/rrqueue.py [hunk 1]
```diff
@@ -11,22 +11,22 @@ class RoundRobinQueue(object):
...
+    previously closed leaving some key buckets non-empty, those keys
     should be passed in start_keys.
 
     The queue maintains a fifo queue of keys.  The key that went last is
     poped first and the next queue for that key is then poped.  This allows
     for a round robin
     """
 
-    def __init__(self, qfactory, start_domains=()):
+    def __init__(self, qfactory, start_keys=()):
```
Comment (Gallaecio) [inline queuelib/rrqueue.py]: @dangra Should we make this backward-compatible?

**Comment by Gallaecio** [comment]
Closing as per https://github.com/scrapy/scrapy/pull/3160

**Pull request closed without merging**